_benchmark/glossary.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: Glossary
+nav_order: 10
+---
+
+# OpenSearch Benchmark glossary
+
+The following terms are commonly used in OpenSearch Benchmark:
+
+- **Corpora**: A collection of documents.
+- **Latency**: If `target-throughput` is disabled (has no value or a value of `0)`, latency is equivalent to service time. If `target-throughput` is enabled (has a value of 1 or greater), latency is the service time plus the time the request waits in the queue before being sent.
+- **Metric keys**: The metrics that OpenSearch Benchmark stores, based on the configuration in the [metrics record]({{site.url}}{{site.baseurl}}/benchmark/metrics/metric-records/).
+- **Operations**: In workloads, a list of API requests performed by a workload.
+- **Pipeline**: A series of steps occurring before and after a workload is run that determines benchmark results.
+- **Schedule**: In workloads, a list of operations in a specific order.
```

Comment:
So, the difference between this and `operations` is that `operations` is not necessarily in order but `schedule` is? Also, is the order time-based, and if so, is it execution time or the time the request was put in the queue or sent?

Reply:
Not exactly. A schedule is simply a list of one or more operations performed in the order they appear at the time the workload is run. The order the operations in a schedule isn't time-based. 

I'll adjust the definition accordingly.